src/modules/previewpane/common/Utilities/StreamWrapper.cs

```diff
@@ -0,0 +1,242 @@
+﻿// Copyright (c) Microsoft Corporation
+// The Microsoft Corporation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.IO;
+using System.Runtime.InteropServices;
+using System.Runtime.InteropServices.ComTypes;
+
+namespace Common.Utilities
+{
+    /// <summary>
+    /// Wraps <see cref="IStream"/> interface into <see cref="Stream"/> Class.
+    /// </summary>
+    /// <remarks>
+    /// Implements only read from the stream functionality.
+    /// </remarks>
+    public class StreamWrapper : Stream

```

Comment:
We should have Unit Tests for the StreamWrapper that verifies `Length`, `Position`, `Read` and `Seek`.

Reply:
Sure i'm planning to cover it in a separate PR.